docs/_sass/_content.scss

```diff
@@ -0,0 +1,41 @@
+.docs {
+  main {
+    hr {
+      margin: 2rem 0 1rem 0;
+    }
+
+    h1 {
+      font-size: 2rem;
```

Comment:
Sorry, I didn't catch that the .docs class wasn't being applied to every template's `<body>`--what you've got here works then.